web/src/hotkey.js

```diff
@@ -370,18 +370,18 @@ function handle_popover_events(event_name) {
         return true;
     }
 
-    if (user_card_popover.user_info_manage_menu_popped()) {
-        user_card_popover.user_info_popover_manage_menu_handle_keyboard(event_name);
+    if (user_card_popover.user_card_manage_menu_popped()) {
+        user_card_popover.user_card_popover_manage_menu_handle_keyboard(event_name);
         return true;
     }
 
-    if (user_card_popover.message_info_popped()) {
-        user_card_popover.user_info_popover_for_message_handle_keyboard(event_name);
+    if (user_card_popover.message_user_card_popped()) {
+        user_card_popover.user_card_popover_for_message_handle_keyboard(event_name);
         return true;
     }
 
-    if (user_card_popover.user_info_popped()) {
-        user_card_popover.user_info_popover_handle_keyboard(event_name);
+    if (user_card_popover.user_card_popped()) {
```

Comment:
I think `user_card_popover.user_info_variant_is_open` might be closer to our standard naming scheme for these. (Sadly bare `is_open` doesn't work here)

Reply:
I guess the general cluster of names around the `user_info_popover_handle_keyboard` may need some thinking on consistency...

Reply:
OK I think after looking at the names in this last commit, they're all better than what we had previously, probably 50% are names that we'll be happy with long term (like the templates), 25% are names for things that may stop needing a name when we do the Tippy rewrite, and the rest are only referenced in <50 lines total, and I'm really not sure what to propose for them that would feel actually good. 

But there's definitely at least one bug (the "manage" menu doesn't work after opening from recent topics with this commit added); just for clarity, that's this little menu:

![image](https://github.com/zulip/zulip/assets/2746074/01a8482c-8ef9-46f1-8ce5-dc2f72122303)

I think a good way to redo the commit structure would be:
* First a commit renaming the templates.
* The a commit renaming `for_user_card_popover`
* Then one commit per individual identifier that appears in the HTML or CSS.
* (The above would cover just about all the changes outside `popover_menus.js` and `hotkey.js`)
* Then probably just one commit for all the `popped` and `handle_keyboard` function renames -- which should be small enough to easily review and cover the changes in `hotkey.js`
* I'm not sure what's else remains.

Another totally fine way, which is probably how I'd do it, is after the templates rename commit, just make the commits on the shell by generating the changes using `sed -i 's/foo/bar/g` to edit the code, `lint --fix` to fix any line-wrapping issues caused by that, and then `git commit -am` to make the individual renaming commit.

Reply:
Renamed `*_popped` functions accordingly:
`user_info_popped` -> `is_user_card_open`
`message_info_popped` -> `is_message_user_card_open`
`user_info_manage_menu_popped` -> `is_user_card_manage_menu_open`

Reply:
Changed the commit structure as you suggested and also added more commits to rename other instances.

Reply:
The bug with "manage menu" is now fixed.